app/components/quiz/QuizAnswer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import AppStore from 'store/appStore';
```

Comment:
this isn't doing anything i can see. but we should be pushing value changes to the answer input into state via the actions --> store.